app/controllers/planets_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PlanetsController < ApplicationController
+  def index
+    @planets = Planet.all
+  end
+
+  def new
+    @planet = Planet.new
+  end
+
+  def create
+    @planet = Planet.new(planets_params)
+    if @planet.save
+      redirect_to planets_path
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def planet_params
+    params.require(:planet).permit(:name, :description, :address, :price, :number_guests)
```

Comment:
Does the number_guests has to be required for a planet? Shouldn't it be for a booking?

Reply:
No, for a planet it is alright. I describes how many guests a planet can welcome